deployment-audit/nmap/cybernet_target_audit.py

```
@@ -0,0 +1,378 @@
+            idx = col_letter_to_index(ref)
+            cell_type = c.attrib.get("t", "")
+            value = ""
+            if cell_type == "inlineStr":
+                value = "".join(t.text or "" for t in c.findall(".//x:t", ns))
+            else:
+                v = c.find("x:v", ns)
+                if v is not None and v.text is not None:
+                    if cell_type == "s":
+                        try:
+                            value = shared[int(v.text)]
+                        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)